exercises/alice_russell/ex02_oo_basics/src/Circle.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Circle.h"
+
+static const double PI = 3.141592653589793238463;
```

Comment:
Good extraction of a constant out to a name and this is good practice to get into generally. Just as note that in Mantid we would use the definition of pi called `M_PI` which you can get from `#include <cmath>`.